uaclient/config.py

```diff
@@ -691,6 +691,15 @@ def _attached_service_status(
             else:
                 ent_status, details = ent.user_facing_status()
 
+        blocked_by = [
+            {
+                "name": service.entitlement.name,
+                "reason_code": service.named_msg.name,
+                "reason": service.named_msg.msg,
+            }
+            for service in ent.blocking_incompatible_services()
+        ]
```

Comment:
I know this is already being tested on the integration tests, but could we have an unittest that touches this too ?

Reply:
Ah yeah we might as well. I'll add it

Reply:
Just pushed this test, LMK what you think

Reply:
LGTM, thanks for that @orndorffgrant